<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>283</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>223</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="number_icons.qrc">
    <normaloff>:/Pobrane/Numbers-1-icon.png</normaloff>:/Pobrane/Numbers-1-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 31, 49);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>26</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="divide">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>95</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_6">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>167</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="changeSign">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>95</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>-/+</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_3">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>203</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>203</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backspace">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>239</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="subtract">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>167</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="comma">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>239</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="multiply">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>131</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>131</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>131</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>167</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_2">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>203</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>203</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="equals">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>239</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_8">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>131</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="percentage">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>95</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_5">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>167</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_0">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>239</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color:white;}
QPushButton:hover{
background-color: rgb(164, 156, 156);
border: 0px;}
QPushButton:pressed {
background-color: rgb(255, 255, 255);
color: black;}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>percentage</tabstop>
  <tabstop>Button_9</tabstop>
  <tabstop>Button_6</tabstop>
  <tabstop>Button_3</tabstop>
  <tabstop>backspace</tabstop>
  <tabstop>equals</tabstop>
  <tabstop>add</tabstop>
  <tabstop>Button_8</tabstop>
  <tabstop>Clear</tabstop>
  <tabstop>changeSign</tabstop>
  <tabstop>Button_0</tabstop>
  <tabstop>Button_4</tabstop>
  <tabstop>subtract</tabstop>
  <tabstop>Button_2</tabstop>
  <tabstop>Button_5</tabstop>
  <tabstop>comma</tabstop>
  <tabstop>Button_1</tabstop>
  <tabstop>multiply</tabstop>
  <tabstop>divide</tabstop>
  <tabstop>Button_7</tabstop>
 </tabstops>
 <resources>
  <include location="number_icons.qrc"/>
 </resources>
 <connections/>
</ui>
